ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -48,16 +48,12 @@ releases:
       relative: 1
       version_bounds:
         lower: 4.16.0-0
-        stream: 4-dev-preview
         upper: 4.17.0-0
   target:
-    prerelease:
+    release:
```

Comment:
From openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml, sounds like it is supposed to use `version_bounds` here too.

Reply:
@jianlinliu ok, i will update accordingly.

Reply:
Just want to know what's the difference between `version_bounds` and `channel`. Maybe a release goes to release controller a bit earlier than cincinnati channel, but not too much time difference.

Reply:
@shellyyang1989 yes, when i tested there was no difference in the output. we can fetch n-1 version using prerelease and relative, whereas in release relative option is not available.

Reply:
Got it. Thank you!